extensions/spring-data-jpa/deployment/src/main/java/io/quarkus/spring/data/deployment/SpringDataJPAProcessor.java

```diff
@@ -58,7 +61,8 @@ void build(CombinedIndexBuildItem index,
                 indexIndex);
 
         removeNoRepositoryBeanClasses(interfacesExtendingCrudRepository);
-        implementCrudRepositories(generatedBeans, additionalBeans, interfacesExtendingCrudRepository, indexIndex);
+        implementCrudRepositories(generatedBeans, additionalBeans, generatedClasses,
```

Comment:
How about we move `generatedClasses` right after `generatedBeans` instead of next to `additionalBeans`. It will make things a tad more consistent I believe.